algorithms/Java/README.md

```diff
@@ -88,6 +88,7 @@
 - [Boyer Moore Search](strings/Boyer_Moore.java)
 - [Reverse String](strings/reverse-string.java)
 - [First Non Repeating Character](strings/first-non-repeating-char.java)
+- [isomprphic Strings](strings/isomorphic_strings.java)
```

Comment:
Please correct the spelling of isomorphic and **I** should be capital.
```suggestion
- [Isomorphic Strings](strings/isomorphic_strings.java)
```

Reply:
Changes were done!